envoy/stream_info/stream_info.h

```diff
@@ -358,13 +358,6 @@ class UpstreamInfo {
    */
   virtual Ssl::ConnectionInfoConstSharedPtr upstreamSslConnection() const PURE;
 
-  /**
-   * Sets the upstream timing information for this stream. This is useful for
-   * when multiple upstream requests are issued and we want to save timing
-   * information for the one that "wins".
-   */
-  virtual void setUpstreamTiming(const UpstreamTiming& upstream_timing) PURE;
-
```

Comment:
This PR seems quite similar to #19118? Does this PR replace that one?

Reply:
yeah, theoretically.  there's 2 issues with that one - the clang tidy which I have fixed locally (not pushed) and the windows build failure, which I could not figure out. I was hoping removing the getters would remove the build failure but left it assigned my way until I was sure.

if it's too big to combine getter and setter clean up, I can likely submit removing getters first, but didn't want to break it up until 1) I knew it fixed the problem and 2) checking to see if you wanted it split or were OK with the one large PR

Reply:
This approach seems totally fine. No need to split. It's a lot of movement but it's largely mechanical/straightforward. It's not like there's lots of complex interactions which are changing, I think.